### PR TITLE
Changed wording of error page

### PR DESCRIPTION
**Pull request by Razzeee**
- Redid wording of error page
- Also redid the order of the items and did a little organization.

Just an idea how it could look, some graphics would also be nice. So please let's get some feedback, as I'm not really an ui guy.

image

### CODE REVIEW

**Comment by yuya-oc** [comment]
Looks better. "What does the user need to check?" would be more helpful than error details.

FYI, I found a issue that `<a>` link doesn't perform. But it seems the issue is not related to this PR.

**Comment by Razzeee** [comment]
Sorry, ment to say that about the link, but somehow forgot. :)

**Comment by jasonblais** [comment]
Looks much better, thanks @Razzeee for your work here!

Propose queuing the proposal in [UX Design channel](https://pre-release.mattermost.com/core/channels/ux-design) where the Mattermost team can offer their feedback.

**Comment by Razzeee** [comment]
This is what the ux team came up with. As I'm on war with CSS I'm pretty sure I will fail at centering everything, but let's see.
screen

**Comment by Razzeee** [comment]
image
So here's a first version with some things still off.
I especially don't like that we don't let the user know that we're refreshing periodically.

**Comment by jasonblais** [comment]
Really nice work here, thanks @Razzeee! 

I'll check with the platform team if they have time to help out with CSS. @asaadmahmood is fantastic with it, so he might be able to do tweaks pretty quickly.

**Comment by Razzeee** [comment]
@asaadmahmood did some look and feel changes, so we're now at this:

image

**Comment by jasonblais** [comment]
Looks great to me. Big thanks @Razzeee and @asaadmahmood! 

We're really improving the UX standard with v1.4, which is awesome.

@yuya-oc: @asaadmahmood worked on this on his Mac, and this works on Windows. Do you think further testing is needed?

File: src/browser/index.jsx [hunk 1]
```diff
@@ -466,29 +466,67 @@ var MattermostView = React.createClass({
...
+        <div style={ errorPage.tableStyle }>
+          <div style={ errorPage.cellStyle }>
+            <Row>
+              <Col xs={ 0 } sm={ 1 } md={ 1 } lg={ 2 } />
+              <Col xs={ 12 } sm={ 10 } md={ 10 } lg={ 8 }>
+              <h2>Cannot connect to Mattermost</h2>
+              <hr />
+              <p>We're having trouble connecting to Mattermost. If refreshing this page (Ctrl+R or Command+R) does not work please verify that:</p>
+              <br />
+              <p>
```
Comment (yuya-oc) [inline src/browser/index.jsx]: The console shows `index.js:1812 Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.` In my setting, which has three teams and the third is incorrect URL, I could not click the second tab while showing the third tab. Is this related?

File: src/browser/index.jsx [hunk 2]
```diff
@@ -466,29 +466,67 @@ var MattermostView = React.createClass({
...
+                <ul style={ errorPage.bullets }>
+                  <li>Your computer is connected to the internet.</li>
+                  <li>The Mattermost URL
```
Comment (yuya-oc) [inline src/browser/index.jsx]: Should add `{' '}`.

File: src/browser/index.jsx [hunk 3]
```diff
@@ -466,29 +466,67 @@ var MattermostView = React.createClass({
 });
 
 // ErrorCode: https://code.google.com/p/chromium/codesearch#chromium/src/net/base/net_error_list.h
-// FIXME: need better wording in English
+var errorPage = {
```
Comment (yuya-oc) [inline src/browser/index.jsx]: Should use `const`.

**Comment by yuya-oc** [comment]
The styling looks fine. Probably this is not related to OS excepting fonts, so I think we need just one tester for next testing.

File: src/browser/index.jsx (hunk 2 above)
Comment (Razzeee) [inline src/browser/index.jsx]: I'm not sure if I understand these two suggestions

File: src/browser/index.jsx [hunk 4]
```diff
@@ -466,29 +466,67 @@ var MattermostView = React.createClass({
...
-        <p>
-          { this.props.errorInfo.errorDescription }
-        </p>
-        <p>Please check below. Then, reload this window. (Ctrl+R or Command+R)</p>
-        <ListGroup>
-          <ListGroupItem>Is your computer online?</ListGroupItem>
-          <ListGroupItem>Is the server alive?</ListGroupItem>
-          <ListGroupItem>Is the URL correct?</ListGroupItem>
-        </ListGroup>
+        <div style={ errorPage.tableStyle }>
```
Comment (yuya-oc) [inline src/browser/index.jsx]: By removing these two `<div>`, tabs properly works while showing the error page. So you might need to adjust the vertical position with other ways.

**Comment by Razzeee** [comment]
Should be fixed now

**Comment by yuya-oc** [comment]
The tabs work properly on Windows 10 and OS X.

@Razzeee Just curious, what was the point in the last push?

**Comment by Razzeee** [comment]
@yuya-oc 
This line https://github.com/mattermost/desktop/pull/254/commits/b294a44e9b11fddcc5409ee35f3b91c2f6e4340c#diff-46044d7460b0a55fc983668900dbd4cbR475
was set to -70 in the code @asaadmahmood delivered. Setting it to 0 fixed it. It might not be 100% in the vertical mid now.

**Comment by yuya-oc** [comment]
I see. Thanks!
